examples/reminders/lib/feature_reminder_list/ui_components/scroll_to_index_widget.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_sticky_header/flutter_sticky_header.dart';
+import 'package:rx_bloc_list/models.dart';
+import 'package:scroll_to_index/scroll_to_index.dart';
+
+import '../../app_extensions.dart';
+import '../../base/common_ui_components/app_reminder_tile.dart';
+import '../../base/common_ui_components/app_sticky_header.dart';
+import '../../base/models/reminder/reminder_model.dart';
+
+class ScrollToPositionWidget extends StatefulWidget {
```

Comment:
The class name `ScrollToPositionWidget` looks very generic, but from what I see this widget can be used only for List of ReminderModel, so please pick more specific name such as `ReminderListScrollView`

Reply:
The name was changed from `ScrollToPositionWidget` to `ReminderListScrollView`